### DISCUSSION TITLE
Allow for custom implementation of span bar colors

### DISCUSSION THREAD

**Discussion by jcolladokuri** [grafana/grafana#62918] [Dashboards, Panels & Visualizations]
Right now, the Traces visualization chooses the span bar colors based on the service name. In our case, we would like to be able to choose a custom mapping, for example:

![image](https://user-images.githubusercontent.com/20448042/216727392-1884b83f-d01f-45ba-818d-00a504219286.png)

in the screenshot above, there is one "style" for requests and another one for dependencies. It would be great if we could recreate this kind of behavior in Grafana!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82859. Feel free to continue the discussion around this there. Thank you!
